test_unstructured/partition/test_api.py

```diff
@@ -140,8 +136,9 @@ def test_partition_via_api_with_no_strategy():
     elements_hi_res = partition_via_api(filename=filename, strategy="hi_res", api_key=get_api_key())
 
     # confirm that hi_res strategy was not passed as default to partition by comparing outputs
-    # FIXME(crag): elements_hi_res[4].text is 'sacon oot barvard o', the fast output.
-    # should be 'Harvard University {melissadell,jacob carlson}@fas.harvard.edu' (as of writing)
+    # elements_hi_res[3].text =
+    #     'LayoutParser: A Uniﬁed Toolkit for Deep Learning Based Document Image Analysis'
+    # while elements_no_strategy[3].text = ']' (as of this writing)
```

Comment:
nit: just noting the index in the comment doesn't match the index in the comparison below, which is slightly awkward.

Reply:
Yup, just changed. Ty